src/data/markdown/docs/30 xk6-browser/01 xk6-browser/03-element-handle.md

```diff
@@ -3,7 +3,48 @@ title: "ElementHandle"
 excerpt: "xk6-browser: ElementHandle Class"
 ---
 
-<BrowserCompatibility/>
+## Supported APIs
```

Comment:
Should we add a warning notice to all of these pages that they're a WIP?

Reply:
I could add the `<BrowserCompatibility/>` back in but that references PW, so maybe something like:

```
<Blockquote mod="warning">

This is still work in progress, and some of the functionality that is listed below may not behave as expected.

</Blockquote>
```

Reply:
Yeah, that text sounds good to me. Just with a slight change: "This documentation is a work in progress, and some ...".

I think we can delete that `BrowserCompatibility` component, we won't use it as is. Or if you prefer to avoid repeating that `Blockquote` on each page, rename the component to something like `BrowserDocsWIP` and use it instead.

Reply:
Resolved in: e075b018e915ff48e97b48d5d8567ee0f0e012cd & c01d309faa84683cc8275f5a2b677b3d4cdf073f